video/probe.go

```diff
@@ -37,7 +37,7 @@ func (p Probe) ProbeFile(url string) (iv InputVideo, err error) {
 	return parseProbeOutput(data)
 }
 
-func parseProbeOutput(probeData *ffprobe.ProbeData) (InputVideo, error) {
+func parseProbeOutput(probeData *ffprobe.ProbeData) (iv InputVideo, err error) {
```

Comment:
Let's not used named returns, since it can make the flow confusing to reason about